mobile_verifier/src/heartbeats.rs

```diff
@@ -84,7 +97,8 @@ impl HeartbeatDaemon {
         file: FileInfoStream<CellHeartbeatIngestReport>,
         cache: &Arc<Cache<(String, DateTime<Utc>), ()>>,
     ) -> anyhow::Result<()> {
-        let epoch = (file.file_info.timestamp - Duration::hours(1))..file.file_info.timestamp;
+        let epoch = (file.file_info.timestamp - Duration::hours(3))
```

Comment:
any special reason this is a 33 hour window or was that intended to be `Duration::hours(3)...Duration::minutes(30)`?

Reply:
Indeed, it is supposed to be 30 minutes